examples/decomposition/plot_pca_vs_fa_model_selection.py

```diff
@@ -58,12 +57,13 @@
 # #############################################################################
 # Fit the models
 
-n_components = np.arange(0, n_features, 5)  # options for n_components
+# options for n_components (45 removed for speed)
+n_components = np.arange(0, n_features - 5, 5)
 
 
 def compute_scores(X):
     pca = PCA(svd_solver='full')
-    fa = FactorAnalysis()
+    fa = FactorAnalysis(tol=1)
```

Comment:
I am not sure it's a good idea to advertise using such a high tolerance without a comment or empirical evidence that it's ok.

my 2c

Reply:
@agramfort Thanks for your comment. I agree! Do you think that the comment introduced in d79e45d is enough? 

I also added the same comment in b742f21 for the `plot_rbm_logistic_classification.py` example where I also increased the tolerance in #13648